ddtrace/compat.py

```diff
@@ -70,6 +70,24 @@ def to_unicode(s):
     return stringify(s)
 
 
+def get_connection_response(conn):
```

Comment:
can we write a regression test for that? I think a real integration test is difficult to write, but you can mock the `getresponse()` and check if `buffering=True` is in the kwargs. This test should be executed only for PY2 since the keyword is not available for PY3.